### PR TITLE
Fix: Fixed issue where delete modal displayed duplicate items

### LINKED ISSUE #10321: Bug: Items in File in use model are shown twice
### Description

image

### Steps To Reproduce

1. Open a file that gets locked by program
2. Attempt to delete it
3. See the file shown twice in the dialog

### Steps to fix

Scan for and display items only one in model

### Files Version

2.3.52.0

### Windows Version

10.0.22621.675

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/9892266/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/9892267/debug_fulltrust.log)

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Items in File in use model are shown twice. Do not add Final to Items of the same instance but to a new instance, like other commands. Otherwise, both Items and Final are displayed. 
Fixes #10321.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by yaichenbaum** [comment]
Can you link to a GitHub issue?

**Comment by cinqmilleans** [comment]
Merge fixed. Fix #10321.

**Comment by yaichenbaum** [comment]
I think the description needs to say `Closes #` or alt` `Fixes #` in order to properly link to the issue.

File: src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs [hunk 1]
```diff
@@ -17,13 +16,11 @@
...
 using System.Threading;
 using System.Threading.Tasks;
-using Windows.ApplicationModel.AppService;
-using Windows.Foundation.Collections;
 using Windows.Storage;
 
 namespace Files.App.Filesystem
 {
-    public class ShellFilesystemOperations : IFilesystemOperations
+	public class ShellFilesystemOperations : IFilesystemOperations
```
Comment (QuaintMako) [inline src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs:23]: Formatting issue.

**Review by QuaintMako — APPROVED** [review]
LGTM.

File: src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs (hunk 1 above)
Comment (QuaintMako) [inline src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs:23]: Forgot that small formatting issue.
```suggestion
    public class ShellFilesystemOperations : IFilesystemOperations
```

**Review by QuaintMako — APPROVED** [review]

**Review by yaichenbaum — COMMENTED** [review]
LGTM

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**